SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -19184,7 +19681,7 @@ void CNSSolver::UpdateAverage(const su2double weight,
 
     /*-- Give Pk a different averaging time than the rest of the terms ---*/
 
-    const su2double new_Pk = production + (current_production - production)*weight*0.25;
+    const su2double new_Pk = production + (current_production - production)*weight*0.5; // for Cave=2
```

Comment:
One of us needs to make this a cfg option.

Reply:
Agreed.